app/javascript/test/game.test.js

```diff
@@ -1,46 +1,65 @@
-import React from 'react';
-import axios from 'axios';
-import MockAdapter from 'axios-mock-adapter';
-import { mount } from 'enzyme';
+import React from "react";
+import axios from "axios";
+import MockAdapter from "axios-mock-adapter";
+import { mount } from "enzyme";
 
-import './setupTests';
+import Game from "../components/game/Game";
 
-import Game from '../components/game/Game';
-
-describe('Game', () => {
+describe("Game", () => {
   let wrapper;
   let mock;
   beforeEach(() => {
-    wrapper = mount(<Game />)
+    wrapper = mount(<Game />);
     mock = new MockAdapter(axios);
+  });
 
-  })
-
-    test('render a h1 welcome message', () => {
-      expect(wrapper.find("h1").text()).toContain("Tic-Tac-Toe");
-    });
+  test("render a h1 welcome message", () => {
```

Comment:
`h1` is an implementation detail here for me. So I would just focus the test description on `render a welcome message`. So if you ever decide to change the welcome message from `h1` to `h2` or any other element, you won't have to change the test description.